src/AssemblerReadGraph2.cpp

```diff
@@ -4,11 +4,143 @@ using namespace shasta;
 
 
 void Assembler::createReadGraph2(
-    uint32_t maxAlignmentCount,
-    uint32_t maxTrim)
+    uint32_t maxAlignmentCount)
 {
-    // This boilerplate code creates the read graph using
-    // all available alignments.
-    vector<bool> keepAlignment(alignmentData.size(), true);
+    // Percentile cutoffs for automated parameters
+    double largePercentile = 0.12;
+    double smallPercentile = 0.015;
+
+    vector<bool> keepAlignment(alignmentData.size(), false);
+
+    // Initialize histograms for measuring alignedFraction, markerCount, maxDrift, and maxSkip distributions
+    Histogram2 alignedFractionHistogram(0, 1, 100);
+    Histogram2 markerCountHistogram(0, 3000, 300);
+    Histogram2 maxDriftHistogram(0, 100, 100);
+    Histogram2 maxSkipHistogram(0, 100, 100);
+    Histogram2 maxTrimHistogram(0, 100, 100);
+
+    ofstream alignmentInfoCsv("alignmentInfo.csv");
```

Comment:
I would use a file name beginning with an uppercase letter for consistentcy with other csv files.

Reply:
Also, avoid "AlignmentInfo" which has another meaning in Shasta.

Reply:
This is actually the values contained in AlignmentInfo, written as a csv. Maybe `AlignmentSummaryStats.csv` is better?

Reply:
Given the contents of this file, I think the name `AlignmentInfo.csv` (without reference to summary or statistics, because this is detailed output) is fine (with uppercase first letter). But like I said in a separate comment, only create it under control of a `const bool debug = false` flag, so you can turn it on easily for debugging if you need to.

Reply:
ok sounds good, it is pretty large, about 3Gb